1694379935757	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1694381101761	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1694381101761	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1694381101764	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1694381101765	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1694381185651	Sync.SyncScheduler	DEBUG	Woke from sleep.
1694381185651	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1694381185651	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1694381185652	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1694381187653	Sync.Service	DEBUG	User-Agent: Firefox/117.0 (Linux x86_64) FxSync/1.119.0.20230828231221.desktop
1694381187653	Sync.Service	INFO	Starting sync at 2023-09-11 06:26:27 in browser session LlnF2-p7N3lB
1694381187654	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1694381187657	Sync.Status	INFO	Resetting Status.
1694381187657	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1694381187657	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1694381187657	FirefoxAccounts	DEBUG	getOAuthToken enter
1694381187658	FirefoxAccounts	TRACE	getCachedToken returning cached token
1694381187658	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1694381187659	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1694381187663	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1694381187664	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]"}
1694381187666	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/198465042/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1694381187666	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1694381187666	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1694381187667	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1694381187667	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1694381187671	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
